src/components/CheckEditor/ProbeOptions.tsx

```diff
@@ -95,7 +96,7 @@ export const ProbeOptions = ({ checkType }: Props) => {
       <Field
         label="Frequency"
         description="How frequently the check should run."
-        disabled={!isEditor || isTraceroute}
+        disabled={!isEditor || isTraceroute} // is this correct?
```

Comment:
I noticed the field is disabled but the input isn't, should frequency be disabled?